lib/puppet/provider/mount/parsed.rb

```diff
@@ -23,7 +23,7 @@
   when "Solaris"
     @fields = [:device, :blockdevice, :name, :fstype, :pass, :atboot, :options]
   else
-    @fields = [:device, :name, :fstype, :options, :dump, :pass]
+    @fields = [:device, :name, :fstype, :options, :dump, :pass, :live_options]
```

Comment:
The implementation of [`Puppet::Provider#set`](https://github.com/puppetlabs/puppet/blob/master/lib/puppet/provider.rb#L543-L547) simply assigns values to `@property_hash`, so we can just call `#set` without having to explicitly add the `live_options` param/value/thing.